HEC.FDA.ViewModel/Saving/SavingBase.cs

```diff
@@ -274,13 +278,13 @@ public void DeleteRowWithCompoundKey(string tableName, int[] primaryKeys, string
             command.ExecuteNonQuery();
         }
 
-        public void DeleteRowWithKey( int key, string keyColName)
+        public void DeleteRowWithKey(int key, string keyColName)
```

Comment:
Not seeing references in this guy. Confirm that. might be able to delete.

Reply:
can confirm, removed references